### ISSUE TITLE
Question about the model selection

### ISSUE THREAD

**Issue by taylover-pei** [easezyc/deep-transfer-learning#6]
Your have done a great job.
But I have a question about the training process.
In the domain adapation scenario, you tune the parameters of the model according to the validation set of source domain or the target domain? 
That means, how do you select the best model? According to the evaluation metrics of source domain validation set or the target domain validation set.
Thank you.

**Comment by easezyc** [comment]
Actually, I consider it is a common problem for domain adaptation community. There are only training and test sets for such as Office31, ImageCLEF, OfficeHome datasets. Hence, I use the test set as unlabeled data for aligning distributions and also select the best models according to the performance on the test set.
